Src/library/base/elks/kernel/type.e

```diff
@@ -120,6 +121,9 @@ feature -- Status report
 
 	is_attached: BOOLEAN
 			-- Is current type attached?
+			--
+			-- Note that this routine does not consider expanded types as attached,
+			-- even in void-safe mode!
```

Comment:
Intuitively we would expect that this routine would return True (when compiled in void-safe mode) when the type handled is expanded. But this is not the case in the current implementation. So, either we document it, as in the added comment above. Or we change the implementation to return True in this case.